Snakefile

```diff
@@ -91,14 +92,16 @@ align_dir = f"{data_dir}/alignment"
 
 # models
 models_dir = f"{data_root_dir}/models/{src}-{trg}/{experiment}"
-teacher_dir = f"{models_dir}/teacher"
+teacher_all_dir = f"{models_dir}/teacher-all"
+teacher_parallel_dir = f"{models_dir}/teacher-parallel"
```

Comment:
From reading the source, I don't understand what `teacher_parallel_dir` should contain. What is a parallel teacher model?

Reply:
Teacher all - the model is trained on all available data.
Teacher parallel - optional model to fine-tune on parallel data only if the data was augmented with back translations.

Will it be easier to understand if I rename them to `teacher` and `teacher-finetuned`?

Reply:
Yes that would be easier to understand, but this is a very minor point.